workshops/views.py

```diff
@@ -201,14 +203,10 @@ def person_bulk_add(request):
     if request.method == 'POST':
         form = PersonBulkAddForm(request.POST, request.FILES)
         if form.is_valid():
+            charset = request.FILES['file'].charset or settings.DEFAULT_CHARSET
+            stream = codecs.getreader(charset)(request.FILES['file']) 
```

Comment:
On Thu, Feb 05, 2015 at 02:19:29PM -0800, Piotr Banaszkiewicz wrote:
> > +            charset = request.FILES['file'].charset or settings.DEFAULT_CHARSET
> > +            stream = codecs.getreader(charset)(request.FILES['file']) 
> 
> **HUUUGE nit-picking**: there's a space left at the end of this line.

Thanks :).  Fixed and force-pushed.